package.json

```diff
@@ -8,27 +8,29 @@
     },
     "workspaces": {
         "packages": [
+            "examples/simple-app",
```

Comment:
Why did we decide to do this in this change?  I think there were some issues I tried it previously... don't remember exactly what they were

Reply:
I can revert it in `alpha` before we cut the proper release. The reason I went ahead with it was that I was having issues with the simple-app when I was testing out the shell changes, and this made it work.